### ISSUE TITLE
Dataset Aligning

### ISSUE THREAD

**Issue by adabrh** [VICO-UoE/Looking3D#7]
Hey.
If i want to know which chair image belongs to which label.

**For Example:**
  ` ShapeNet\ {c (class)} \ {label} `

Here Our Class is **chair**. I want to know how to align **labels** of shapenet with brokenchair dataset.
Kindly help in this matter.

**Comment by ankanbhunia** [comment]
The shapenet id is available in the model.txt file for each shape folder. 

see: [this line in the README](https://github.com/VICO-UoE/Looking3D#:~:text=%5B3%5DBrokenChairs/shapes/%3A%20%3Cmv_images/*.png%3E%3A%20grayscale%20multi%2Dview%20image%2C%20%3Cmv_images/*.json%3E%3A%20json%20file%20containing%20intristic%20and%20extrinsic%20parameters%20of%20the%20rendered%20image%2C%20%3Cmv_images/*.npy%3E%3A%20npy%20file%20containing%202D%2D3D%20correspondence%20points.%20%3Cmodel_id.txt%3E%3A%20corresponding%20ShapeNet%20id.)

**Comment by adabrh** [comment]
Okay. Thanks a lot.
Can you suggest any future work regarding this research?

**Comment by adabrh** [comment]
Hey. @ankanbhunia 
visualize.py file shows a lot of errors while running experiment.py file. Am running it because i want to visualize results on trained model. Can you please re share it?

As you can see in line No. 182 of [visualize.py](https://github.com/VICO-UoE/Looking3D/blob/main/utils/visualize.py)
`imgs = ((batch['imgs'].permute(0,2,3,1).detach().cpu().numpy()+1)/2)*255`
it is accessing key '**imgs**' in **batch** dictionary. 

But in line No. 261 of [dataset_e2e.py](https://github.com/VICO-UoE/Looking3D/blob/main/data/dataset_e2e.py)
`return_dict = {'query_imgs':query_imgs, 
                           'mesh_images':mesh_images, 
                           'mesh_pos_enc3d': mesh_pos_enc3d,
                           'labels':labels, 
                           'bbox':bbox, 
                           'img_v1':img_v1, 
                           'img_v2':img_v2, 
                           'pxy_v1':pxy_v1, 
                           'pxy_v2':pxy_v2, 
                           'img_c':img_c, 
                           'path':img_path}`

There is no **imgs** key in returning dict.
There are also a lot more errors. Could you guide to a solution free Error? Thank You.
